plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java

```diff
@@ -690,10 +693,13 @@ private class OpenContextJob extends AbstractJob {
         private final DBSInstance instance;
         private final Runnable onSuccess;
         private Throwable error;
-        OpenContextJob(DBSInstance instance, Runnable onSuccess) {
+        private boolean readFromInstance;
+
+        OpenContextJob(DBSInstance instance, Runnable onSuccess, boolean readFromInstance) {
```

Comment:
As for me, it's not clear what is this argument about by the name `readFromInstance`, like read what? I'd call it `readContextDefaultsFromInstance` or `readDefaultsFromInstance`. Yes, it's longer, but clearer.

Reply:
ok